docker-java-api/src/main/java/com/github/dockerjava/api/command/StatsCmd.java

```diff
@@ -16,6 +16,11 @@
 
     StatsCmd withContainerId(@Nonnull String containerId);
 
+    @CheckForNull
+    Boolean getNoStream();
```

Comment:
please follow the naming of other boolean properties as seen in other commands (e.g. `AttachContainerCmd`)

Reply:
Renamed to hasNoStream